pipeline-model-definition/pom.xml

```diff
@@ -158,6 +164,12 @@
       <version>1.1.1</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.jenkins.docker</groupId>
+      <artifactId>docker-plugin</artifactId>
+      <version>1.2-SNAPSHOT</version>
```

Comment:
Use the time stamped snapshot version here.

Reply:
correct me if I'm wrong, but we don't publish timestamped snapshots for plugins' pull-request builds
https://ci.jenkins.io/job/Plugins/job/docker-plugin/job/JENKINS-48050/8/console

Reply:
Correct. You need to do a mvn deploy by hand. Also, make sure these new dependencies are in dependencyManagement in the parent pom.